src/test/java/org/mifos/integrationtest/cucumber/stepdef/ValidateChannelDef.java

```diff
@@ -50,29 +50,29 @@ public void iCanCreateATransactionChannelRequestDTOWithNoPayee() {
 
     @When("I call the post transfer API with expected status of {int}")
     public void iCallThePostTransferAPIWithExpectedStatusOf(int expectedStatus) {
-            RequestSpecification requestSpec = Utils.getDefaultSpec();
-            scenarioScopeState.tenant = tenant;
-            scenarioScopeState.response = RestAssured.given(requestSpec).header("Content-Type", "application/json")
-                    .header(tenantHeader, scenarioScopeState.tenant).baseUri(channelConnectorConfig.channelConnectorContactPoint)
-                    .body(scenarioScopeState.createTransactionChannelRequestBody).expect()
-                    .spec(new ResponseSpecBuilder().expectStatusCode(expectedStatus).build()).when()
-                    .post(channelConnectorConfig.transferEndpoint).andReturn().asString();
-
-            logger.info("Post Transfer Response: {}", scenarioScopeState.response);
+        RequestSpecification requestSpec = Utils.getDefaultSpec();
+        scenarioScopeState.tenant = tenant;
+        scenarioScopeState.response = RestAssured.given(requestSpec).header("Content-Type", "application/json")
+                .header(tenantHeader, scenarioScopeState.tenant).baseUri(channelConnectorConfig.channelConnectorContactPoint)
+                .body(scenarioScopeState.createTransactionChannelRequestBody).expect()
+                .spec(new ResponseSpecBuilder().expectStatusCode(expectedStatus).build()).when()
+                .post(channelConnectorConfig.transferEndpoint).andReturn().asString();
+
+        logger.info("Post Transfer Response: {}", scenarioScopeState.response);
     }
 
     @When("I call the transaction request API with expected status of {int}")
     public void iCallTheTransactionRequestAPIWithExpectedStatusOf(int expectedStatus) {
-            RequestSpecification requestSpec = Utils.getDefaultSpec();
+        RequestSpecification requestSpec = Utils.getDefaultSpec();
 
-            scenarioScopeState.tenant = tenant;
-            scenarioScopeState.response = RestAssured.given(requestSpec).header("Content-Type", "application/json")
-                    .header(tenantHeader, scenarioScopeState.tenant).baseUri(channelConnectorConfig.channelConnectorContactPoint)
-                    .body(scenarioScopeState.createTransactionChannelRequestBody).expect()
-                    .spec(new ResponseSpecBuilder().expectStatusCode(expectedStatus).build()).when()
-                    .post(channelConnectorConfig.transferReqEndpoint).andReturn().asString();
+        scenarioScopeState.tenant = tenant;
+        scenarioScopeState.response = RestAssured.given(requestSpec).header("Content-Type", "application/json")
```

Comment:
use constants

Reply:
This files were changed because of spotless those are not new feature introduced by me